assets/findify-autocomplete.js

```diff
@@ -247,11 +255,11 @@ const initAutocomplete = () => {
             };
             initializeFindifyAutocomplete();
     } 
-
+  
     initializeDropdownAutocomplete();
      */
-
+  
     initializeFullscreenAutocomplete();  
-}
-
-setTimeout(() => initAutocomplete(), 500); // wait for findify to load
\ No newline at end of file
+  }
+  
+  setTimeout(() => initAutocomplete(), 500); // wait for findify to load
```

Comment:
For this one I have opened for you a bug.
I think you can enough quickly integrate it in this PR.

Within ticket I have left a comment, I have done something similar within search-block